src/main/java/gregtech/api/recipes/RecipeBuilder.java

```diff
@@ -119,21 +119,29 @@ public R cleanroom(@Nullable CleanroomType cleanroom) {
         return (R) this;
     }
 
-    public boolean applyProperty(@Nonnull String key, @Nullable Object value) {
+    public boolean applyPropertyCT(@Nonnull String key, @Nullable Object value) {
         if (key.equals(CleanroomProperty.KEY)) {
             if (value instanceof CleanroomType) {
                 this.cleanroom((CleanroomType) value);
+                return true;
             } else if (value instanceof String) {
                 this.cleanroom(CleanroomType.getByName((String) value));
-            } else {
-                return false;
+                return true;
             }
-            return true;
+            return false;
         }
         return false;
     }
 
-    public boolean applyProperty(@Nonnull RecipeProperty<?> property, @Nullable Object value) {
+    /**
+     * @deprecated use {@link RecipeBuilder#applyPropertyCT(String, Object)} instead
+     */
+    @Deprecated
+    public boolean applyProperty(@Nonnull String key, @Nullable Object value) {
```

Comment:
Since this PR has stalled for a bit, something has been added that calls this deprecated method, which needs to be switched to the new method.